api/v1alpha1/obtenant_types.go

```diff
@@ -50,6 +50,7 @@ type OBTenantSpec struct {
 	TenantRole  apitypes.TenantRole `json:"tenantRole,omitempty"`
 	Source      *TenantSourceSpec   `json:"source,omitempty"`
 	Credentials TenantCredentials   `json:"credentials,omitempty"`
+	Scenario    string              `json:"scenario,omitempty"`
```

Comment:
Why does not tenant need default optimization scenario?

Reply:
tenant will use the scenario of the obcluster it belongs to if it's not set, this logic is implemented in the tenant's webhook